README.md

```diff
@@ -1,79 +1,119 @@
-# MeiliDB
+<p align="center">
+  <a href="https://www.meilisearch.com">
+    <img src="https://www.meilisearch.com/assets/logo-59b9e6f726360eccac4b0cae0e268ef29b64fc9d386ad4a4e3c10c3e02240ade.svg" width="318px" alt="Strapi logo" />
+  </a>
+</p>
+<h3 align="center">Instant search API made simple.</h3>
+<p align="center">A instant search engine, working out-of-the-box for user-friendly search experience.
 
-[![Build Status](https://dev.azure.com/thomas0884/thomas/_apis/build/status/meilisearch.MeiliDB?branchName=master)](https://dev.azure.com/thomas0884/thomas/_build/latest?definitionId=1&branchName=master)
-[![dependency status](https://deps.rs/repo/github/meilisearch/MeiliDB/status.svg)](https://deps.rs/repo/github/meilisearch/MeiliDB)
-[![License](https://img.shields.io/badge/license-commons%20clause-lightgrey)](https://commonsclause.com/)
+</p>
+<br />
+<p align="center">
+  <a href="https://dev.azure.com/thomas0884/thomas/_build/latest?definitionId=1&branchName=master">
+    <img src="https://dev.azure.com/thomas0884/thomas/_apis/build/status/meilisearch.MeiliDB?branchName=master" alt="NPM Version" />
+  </a>
+  <a href="https://deps.rs/repo/github/meilisearch/MeiliDB">
+    <img src="https://deps.rs/repo/github/meilisearch/MeiliDB/status.svg" alt="Monthly download on NPM" />
+  </a>
+  <a href="https://commonsclause.com/">
+    <img src="https://img.shields.io/badge/license-commons%20clause-lightgrey" alt="Travis Build Status" />
+  </a>
+</p>
 
-A _full-text search database_ based on the fast [LMDB key-value store](https://en.wikipedia.org/wiki/Lightning_Memory-Mapped_Database).
+____
```

Comment:
```suggestion
-------
```

Reply:
And with the other changes requested, I will probably please you to remove that line